trunk/src/modules/paraview.F90

```diff
@@ -14,7 +14,10 @@ module paraview
    integer,parameter  :: PARAVIEW_IO = 22
    logical            :: PARAVIEW_DUMP_GEOM = .FALSE.
    logical            :: PARAVIEW_DUMP_ATTR = .FALSE.
-   logical            :: SECOND_ORDER_VIS   = .FALSE.
+   logical            :: SECOND_ORDER_VIS   = .TRUE.
```

Comment:
I think linear outputting as a default is preferable (often, meshes are just rectilinear).

Reply:
Changes implemented: Linear output is the default one.